playbooks/ORCHESTRATION.md

```diff
@@ -242,13 +242,15 @@ Submitting this PR will trigger a push to the preproduction GCS bucket, so ensur
 
 ## Post-ceremony Steps
 
-1. If any root keyholders have changed, update the [current root keyholders](https://github.com/sigstore/root-signing#current-sigstore-root-keyholders) with their name, key ID, and location of their key material.
+1. The preproduction GCS bucket will need to be manually synced to the GCS production bucket as of [916](https://github.com/sigstore/root-signing/pull/916).
```

Comment:
We can do this as a follow up, but we should link to the GHA (which doesn't yet exist)